### ISSUE TITLE
多标签预测scores为什么大于1？

### ISSUE THREAD

**Issue by suhangbo** [PaddlePaddle/PaddleClas#2113]
多标签infer结果，scores有些值怎么大于1了，求大神指教
![image](https://user-images.githubusercontent.com/52941676/176338244-b35e5ac1-8a83-40ec-9012-9e2d087d20b9.png)

**Comment by cuicheng01** [comment]
hi，请问您执行的是哪个命令呢？

**Comment by suhangbo** [comment]
> hi，请问您执行的是哪个命令呢？

CUDA_VISIBLE_DEVICES=0,1,2,3 python  -m paddle.distributed.launch --gpus 0,1,2,3 --log_dir=output/nsfw/PPLCNet_x2_5_nsfw-xtc_momentum_multi/infer  tools/infer.py -c ppcls/configs/nsfw/PPLCNet_x2_5_nsfw-xtc_momentum_multi.yaml -o Global.pretrained_model=output/nsfw/PPLCNet_x2_5_nsfw-xtc_momentum_multi/PPLCNet_x2_5/epoch_120 &> output/nsfw/PPLCNet_x2_5_nsfw-xtc_momentum_multi/infer/infer.log 2>&1 &

**Comment by cuicheng01** [comment]
麻烦版本也提供一下吧，目前没能复现这个问题。

**Comment by suhangbo** [comment]
> 麻烦版本也提供一下吧，目前没能复现这个问题。

使用的版本为 release/2.3

**Comment by suhangbo** [comment]
> 麻烦版本也提供一下吧，目前没能复现这个问题。

hi,问题已解决，是因为我们在进行单标签预测时，注释了 x = F.softmax(x, axis=-1) if not multilabel else F.sigmoid(x)，输出的原值，在进行多标签预测时未进行sigmoid()处理

**Comment by cuicheng01** [comment]
嗯，好的
